fix: [filament_view] compiler warning

```
[151/192] Building CXX object plugins/filament_view/CMakeFiles/plugin_filament_view.dir/core/scene/view_target.cc.o
/mnt/raid10/workspace-automation/app/ivi-homescreen-plugins/plugins/filament_view/core/scene/view_target.cc:562:11: warning: unused variable 's_lastTime' [-Wunused-variable]
  562 |   int32_t s_lastTime = m_LastTime;  // NOLINT
      |           ^~~~~~~~~~
1 warning generated.
```

Consider warn as error turned on ;)

--- a/plugins/filament_view/core/scene/view_target.cc
+++ b/plugins/filament_view/core/scene/view_target.cc
@@ -558,8 +558,6 @@ void ViewTarget::DrawFrame(const uint32_t time) {
   //   std::chrono::duration<double, std::milli>(cpuUpdateDuration).count()
   // );
 
-  // cast time to int32_t, since EncodableValue does not support uint32_t
-  int32_t s_lastTime = m_LastTime;  // NOLINT
   // TODO(kerberjg): send kUpdateFrame event, async with wait
 
   // Render the scene, unless the renderer wants to skip the frame.

